### PR TITLE
Setup Arteria CheckQC

### PR DESCRIPTION
**Pull request by Aratz**
This PR sets up the CheckQC webservice on Miarka.

### CODE REVIEW

File: roles/arteria-checkqc-ws/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - include: install_config.yml
+
+  - name: add static folders to be created
+    set_fact:
+      static_folders: "{{ static_folders + [arteria_service_log_dir, arteria_service_log_dir + '/nxf_logs', runfolder_path] }}"
+
+  - name: Store {{ arteria_service_name }} tool version in deployment
+    lineinfile:
+      dest: "{{ deployed_tool_versions }}"
+      line: "{{ arteria_service_name }}: {{ seqreport_service_version }}"
```
Comment (b97pla) [inline roles/arteria-checkqc-ws/tasks/main.yml:14]:
```suggestion
      line: "{{ arteria_service_name }}: {{ checkqc_version }}"
```

File: roles/arteria-checkqc-ws/tasks/install.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+
+- name: create virtual python env for sequencing report service
```
Comment (b97pla) [inline roles/arteria-checkqc-ws/tasks/install.yml:3]: nitpick but it's not "sequencing report service"

File: roles/arteria-checkqc-ws/templates/checkqc_logger.config.j2 [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+handlers:
+    console:
+        class: logging.StreamHandler
+        level: DEBUG
+        formatter: simple
+        stream: ext://sys.stdout
+
+    file_handler:
+        class: logging.handlers.RotatingFileHandler
+        level: WARNING  # By default set to WARNING, but can be changed on runtime
```
Comment (b97pla) [inline roles/arteria-checkqc-ws/templates/checkqc_logger.config.j2:19]: perhaps better to use `INFO`?

**Review by b97pla — CHANGES_REQUESTED** [review]
:+1: nice work, just a minor issue

**Comment by Aratz** [comment]
Nice catch! Should be fixed now

**Review by b97pla — APPROVED** [review]
:+1:

**Pull request merged**